manifests/init.pp

```diff
@@ -133,4 +135,24 @@
     links  => follow,
     notify => $notify_services,
   }
+
+  if $ensure == 'present' and $hwutc != undef {
+    $hwclock_cmd = lookup('timezone::hwclock_cmd', Optional[String], 'first', undef)
+    if ($hwclock_cmd == '') or ($hwclock_cmd == undef) { fail("")}
```

Comment:
Keeping the lookup before without a default parameter will give us this validation for free.